docs/server/source/quickstart.md

```diff
@@ -14,7 +14,7 @@ $ sudo mongod --replSet=bigchain-rs
 C. Ubuntu 16.04 already has Python 3.5, so you don't need to install it, but you do need to install some other things:
 ```text
 $ sudo apt-get update
-$ sudo apt-get install g++ python3-dev libffi-dev
+$ sudo apt-get install g++ python3-dev libffi-dev build-essential libssl-dev
```

Comment:
The same change should also be made in the docs page with source at:

`bigchaindb/docs/server/source/appendices/install-os-level-deps.md`

(I don't know if the base requirements on Fedora have changed but I'm not going to worry about that. Folks running it know what to do.)